apps/browser/components/Chat.tsx

```diff
@@ -19,182 +25,100 @@ export interface ChatLog {
 }
 
 export interface ChatProps {
-  logs: ChatLog[];
-  samplePrompts: ExamplePrompt[];
-  isRunning: boolean;
-  isStopped: boolean;
-  isPaused: boolean;
-  isSending: boolean;
-  onPause: () => void;
-  onContinue: () => void;
-  onPromptSent: (prompt: string) => Promise<void>;
+  isAuthenticated: boolean;
```

Comment:
Is this prop needed if we have the `useSession` hook? Since we're already using global state hooks here

Reply:
Figured this was simpler than using the `useSession` + `session === "authenticated"` within another component.